index.js

```diff
@@ -1,20 +1,440 @@
-const readline = require('readline');
-const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+
+/* Notes on project status:
+- I need to rework my if/else if/else statements, because the logic is getting jumbled;
+- I think I need to create more distinction between how I use my rooms and how I use my items;
+- Presently, my currentRoom variable is not getting updated when the player enters a new room and I'm not sure as to why;
```

Comment:
In your `changeRoom` method, you never assign a new value to `playerObject.currentRoom`. This would be why it isn't updating; I'm assuming this is because the code was written before you implemented the player object.